### DISCUSSION TITLE
视频文件好像被禁了，其他文件都能正常访问

### DISCUSSION THREAD

**Discussion by YOU-Y8** [messense/aliyundrive-webdav#762] [Q&A]
视频文件不能复制到本地，不能打开。但是字幕文件或者文档之类的都能正常打开或者复制到本地。目前是2.0.4版。有一样的问题的吗？

**Comment by luoxuele** [comment]
我一直有这样的问题，部分文件能打开，部分打不开，而且不仅仅是视频，连pdf都有这样的问题

**Reply by luoxuele** [reply]
我直接往桌面拖动复制，也是部分文件可以，部分不可以，还是同一个文件夹，莫名其妙

**Comment by hzl88688** [comment]
视频文件没禁，禁止了音乐文件 如MP3 FLAC APE等文件不能直链下载了
